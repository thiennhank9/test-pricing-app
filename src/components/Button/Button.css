button {
  height: 30px;
  width: 100px;
  font-size: 15px;
}
