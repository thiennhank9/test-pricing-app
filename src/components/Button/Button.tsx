import React from "react";
import "./Button.css";

function Button(props: React.ButtonHTMLAttributes<HTMLButtonElement>) {
  return <button {...props} />;
}

export default Button;
