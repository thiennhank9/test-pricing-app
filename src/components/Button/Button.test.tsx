import TestRenderer from "react-test-renderer";
import Button from "./Button";

describe("Button", () => {
  test("render button", () => {
    const testRenderer = TestRenderer.create(<Button />);
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });
});
