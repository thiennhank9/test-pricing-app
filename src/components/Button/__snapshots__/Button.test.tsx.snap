// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button render button 1`] = `<button />`;
